### PR TITLE
ZEPPELIN-467 Fix ace editor focusing problem in chrome

### PR DESCRIPTION
**Pull request by Leemoonsoo**
This PR addresses https://issues.apache.org/jira/browse/ZEPPELIN-467

When (invisible) textarea position is not correctly placed(this problem occur in Chrome when contents in editor is long), place it in cursor position of ace editor. not always set to 0.

This PR also includes scrolling animation improvement. Make animation faster (duration 200 -> 100ms), and cancel animation before new animation starts.

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Ready to merge

File: zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js [hunk 1]
```diff
@@ -698,7 +702,9 @@ angular.module('zeppelinWebApp')
         scrollTargetPos = documentHeight;
       }
     }
-    angular.element('body').scrollTo(scrollTargetPos, {axis: 'y', interrupt: true, duration:200});
+    angular.element('body').stop();
+    angular.element('body').finish();
+    angular.element('body').scrollTo(scrollTargetPos, {axis: 'y', interrupt: true, duration:100});
```
Comment (r-kamath) [inline zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js]: @Leemoonsoo avoid using `angular.element('body')` multiple times. better to have it in a `var` and reuse.

**Comment by Leemoonsoo** [comment]
@r-kamath removed repeating code.

**Comment by r-kamath** [comment]
:+1: LGTM. works well on Chrome and Firefox

**Comment by Leemoonsoo** [comment]
Thanks @r-kamath for review.
Merge if there're no more discussions

**Pull request closed without merging**